include/highfive/bits/H5Utils.hpp

```diff
@@ -278,19 +281,18 @@ inline std::vector<std::size_t> to_vector_size_t(const std::vector<std::size_t>&
 // read name from a H5 object using the specified function
 template<typename T>
 inline std::string get_name(T fct) {
-    const size_t maxLength = 255;
-    char buffer[maxLength + 1];
-    ssize_t retcode = fct(buffer, static_cast<hsize_t>(maxLength) + 1);
-    if (retcode < 0) {
-        HDF5ErrMapper::ToException<GroupException>("Error accessing object name");
-    }
-    const size_t length = static_cast<std::size_t>(retcode);
-    if (length <= maxLength) {
-        return std::string(buffer, length);
-    }
-    std::vector<char> bigBuffer(length + 1, 0);
-    fct(bigBuffer.data(), static_cast<hsize_t>(length) + 1);
-    return std::string(bigBuffer.data(), length);
+  char buffer[H5_MAX_PATH_LEN + 1];
+  ssize_t retcode = fct(buffer, static_cast<hsize_t>(H5_MAX_PATH_LEN) + 1);
+  if (retcode < 0) {
+    HDF5ErrMapper::ToException<GroupException>("Error accessing object name");
+  }
+  const size_t length = static_cast<std::size_t>(retcode);
+  if (length <= H5_MAX_PATH_LEN) {
+    return std::string(buffer, length);
+  }
+  std::vector<char> bigBuffer(length + 1, 0);
+  fct(bigBuffer.data(), static_cast<hsize_t>(length) + 1);
+  return std::string(bigBuffer.data(), length);
```

Comment:
```suggestion
    ssize_t retcode = fct(buffer, static_cast<hsize_t>(H5_MAX_PATH_LEN) + 1);
    if (retcode < 0) {
        HDF5ErrMapper::ToException<GroupException>("Error accessing object name");
    }
    const size_t length = static_cast<std::size_t>(retcode);
    if (length <= H5_MAX_PATH_LEN) {
        return std::string(buffer, length);
    }
    std::vector<char> bigBuffer(length + 1, 0);
    fct(bigBuffer.data(), static_cast<hsize_t>(length) + 1);
    return std::string(bigBuffer.data(), length);
```

Reply:
I also hate indentation inequality, but there are so many files that suffer from this so I think there should be another PR aimed to fix indentations :)

Reply:
The indentation size for this project is supposed to be `4` 
https://github.com/BlueBrain/HighFive/blob/90e6316c171861fb925e7f995533114a4d206614/.clang-format#L60
If it is not `4` somewhere it should be considered a mistake and indeed corrected (in a separate PR like you suggest)

Reply:
Oh thank you, had no thoughts that there are rules for formating projects. Good to know